doc/mgr/restful.rst

```diff
@@ -77,6 +77,21 @@ If the port is not configured, *restful* will bind to port ``8003``.
 If the address it not configured, the *restful* will bind to ``::``,
 which corresponds to all available IPv4 and IPv6 addresses.
 
+Configuring the max_request
+---------------------------
+
+The maximum request size can be configured via the configuration key
+facility::
+
+  ceph config set mgr mgr/restful/$name/max_requests $NUM
+
+where ``$name`` is the ID of the ceph-mgr daemon (usually the hostname).
+by default, the max request number is 5000.
```

Comment:
s/5000/500/ . but i think we can instead always use something like 

```
:confval:`mgr/restful/max_requests`
```

to reference a confval, and use 
```
.. mgr_module:: restful
....
.. confval:: max_requests
```

to declare a confval. please see other mgr modules for more example. less repeating this way.

Reply:
Done, thanks!

Reply:
@NitzanMordhai could you please re-read my suggestion? it's not done. see https://ceph--54984.org.readthedocs.build/en/54984/mgr/restful/

Reply:
@NitzanMordhai please reference https://raw.githubusercontent.com/ceph/ceph/main/doc/mgr/influx.rst . again, 

add at the beginning of the doc
```
.. mgr_module:: restful
```

add this at the place where we put all options:
```
.. confval:: max_requests
```

put this at the place where reference `max_requests`
```
:confval:`mgr/restful/max_requests`
```